doc/book/api.md

```diff
@@ -0,0 +1,186 @@
+API
+===
```

Comment:
I prefer to use `# API` instead for to have a common syntax in the whole document